defaults/main.yml

```diff
@@ -1,20 +1,66 @@
-# Defaults variables for role duffy
-# Duffy itself is on github, this role is used to just deploy and update it on server
-
-# MySQL settings
-duffy_mysqldb_name: duffy
-duffy_mysqldb_user: duffy
-duffy_mysqldb_pass: EtCMT4wdP77avst96M # of course change this for real deploy through inventory, just default generated one
-
-# Duffy Application settings
-# Unprivileged user that will run duffy
-duffy_user: duffy
-#where to create duffy python venv
-duffy_python_venv_path: /var/lib/duffy-venv/
-# where to deploy duffy code
-duffy_app_basedir: /var/lib/duffy
-# upstream duffy git repo
-duffy_upstream_git_repo: https://github.com/CentOS/duffy.git
-# duffy version/branch/commit/tag to use
-duffy_git_version: master
+---
+# Defaults variables for the duffy role
 
+postgresql_client_pkgs:
+  - postgresql
+  - libpq
+
+duffy_python_executable: python3
+duffy_python_pkgs:
+  - python3
+  - python3-devel
+  - python3-pip
+  - python3-setuptools
+duffy_dev_pkgs:
+  - gcc
+  - libpq-devel
+duffy_venv: /var/lib/duffy/venv
+duffy_tasks_schedule_filename: /var/lib/duffy/celerybeat-schedule
+duffy_config_files:
+  - 10_app
+  - 20_metaclient
+  - 30_logging
+  - 40_tasks
+  - 50_database
+  - 60_misc
+  - 70_nodepools
+  - 80_secrets
+
+# TCP ports to open up in the firewall
+duffy_tcp_ports:
```

Comment:
no need to declare variables twice ? if `duffy_app_port` and `duffy_metaclient_port` are defined, let's just use that directly to call iptables roles ?

Reply:
Can I reuse variables like that in the vars file?